topics/basics/prepared_statements/README.md

```diff
@@ -1,11 +1,11 @@
 # Prepared Statements
 
-Prepared statements can be used to improve the performance of frequently executed queries. Preparing the query caches it on the Cassandra cluster and only needs to be performed once. Once created, prepared statements should be reused with different bind variables. Prepared queries use the `?` marked to denote bind variables in the query string.
+Prepared statements can be used to improve the performance of frequently executed queries. Preparing the query caches it on the Cassandra cluster and only needs to be performed once. Once created, prepared statements should be reused with different bind variables. Prepared queries use the `?` marker to denote "anonymous" bind variables in the query string. You can also use named bind variables if you specify them as `:name`.
```

Comment:
nit: "anonymous" => anonymous; I don't think the quotes are necessary here

Reply:
I wouldn't call it anonymous at all. The names can often be determined by the query itself.

`SELECT * FROM test WHERE foo = ?`

The marker `?` can be determined by the name `foo` in the query string for both simple and prepared statements without having to explicitly name the marker. The applies to `INSERT` and `UPDATE` queires as well.